plugins/vault/src/components/EntityVaultTable/EntityVaultTable.tsx

```diff
@@ -54,7 +54,7 @@ export const EntityVaultTable = ({ entity }: { entity: Entity }) => {
 
   const data = (value || []).map(secret => {
     return {
-      secret: secret.name,
+      secret: `${secret.path.replace(secretPath + "/", "")}/${secret.name}`,
```

Comment:
To keep consistency, I would also use this value for the icon Link's `title` below.

Reply:
Sure! I forgot about the titles

Reply:
Cool. So in my opinion this is now ready. @benjdlambert what do you think?

Reply:
Can we maybe extract this out into a variable and re-use the variable rather than doing it mulitple times inside the function?

Reply:
I've created a function to generate the string and re-use it

Reply:
I switched to using a variable. It seemed more appropriate :)